packages/eslint-config-office-addins/package.json

```diff
@@ -16,6 +16,10 @@
   "dependencies": {
     "eslint-plugin-office-addins": "^0.2.2"
   },
+  "peerDependencies": {
```

Comment:
why is this needed? I thought office-addin-lint as a dependency was enough.

Reply:
Or perhaps the peer dependency should be on office-addin-lint...

Reply:
The config package needs to be deprecated since it's redundant.  All it's doing right now is point to the config information in the plugin package.  It's not using the linter package and so there is not dependency for it.